server/controllers/events/events_controller_test.go

```diff
@@ -189,6 +189,7 @@ func TestPost_GithubCommentInvalidCommand(t *testing.T) {
 	w := httptest.NewRecorder()
 	e.Post(w, req)
 	ResponseContains(t, w, http.StatusOK, "Ignoring non-command comment: \"\"")
+	vcsClient.VerifyWasCalled(Never()).ReactToComment(models.Repo{}, 1, "eyes")
```

Comment:
Hmmm although this is a "successful" test in that it works, I don't know if this is the best test for this.

How about the test `TestPost_GithubCommentResponse` to see the reaction ?

https://github.com/runatlantis/atlantis/blob/41310bf8d1a0ea2426ed38a52cf0ef275b2927d6/server/controllers/events/events_controller_test.go#L335

Please see the other tests too. There may be others or more applicable ones for the line you added. If a new test needs to be created, that's also acceptable.